OracleDatabase/RAC/OracleConnectionManager/dockerfiles/21.3.0/createCman.sh

```diff
@@ -0,0 +1,6 @@
+/usr/bin/docker run -d --hostname racnode-cman1 --dns-search=internal.us.oracle.com \
```

Comment:
This script is missing a copyright header and the shebang line. It also has an internal Oracle hostname that must be removed.